contracts/zero-ex/contracts/src/features/MetaTransactionsFeatureV2.sol

```diff
@@ -0,0 +1,620 @@
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-erc20/src/IEtherToken.sol";
+import "@0x/contracts-utils/contracts/src/v06/errors/LibRichErrorsV06.sol";
+import "@0x/contracts-utils/contracts/src/v06/LibBytesV06.sol";
+import "@0x/contracts-utils/contracts/src/v06/LibSafeMathV06.sol";
+import "../errors/LibMetaTransactionsRichErrors.sol";
+import "../fixins/FixinCommon.sol";
+import "../fixins/FixinReentrancyGuard.sol";
+import "../fixins/FixinTokenSpender.sol";
+import "../fixins/FixinEIP712.sol";
+import "../migrations/LibMigrate.sol";
+import "../storage/LibMetaTransactionsStorage.sol";
+import "./interfaces/IFeature.sol";
+import "./interfaces/IMetaTransactionsFeatureV2.sol";
+import "./interfaces/IMultiplexFeature.sol";
+import "./interfaces/INativeOrdersFeature.sol";
+import "./interfaces/ITransformERC20Feature.sol";
+import "./libs/LibSignature.sol";
+
+/// @dev MetaTransactions feature.
+contract MetaTransactionsFeatureV2 is
+    IFeature,
+    IMetaTransactionsFeatureV2,
+    FixinCommon,
+    FixinReentrancyGuard,
+    FixinEIP712,
+    FixinTokenSpender
+{
+    using LibBytesV06 for bytes;
+    using LibRichErrorsV06 for bytes;
+
+    /// @dev Describes the state of a meta transaction.
+    struct ExecuteState {
+        // Sender of the meta-transaction.
+        address sender;
+        // Hash of the meta-transaction data.
+        bytes32 hash;
+        // The meta-transaction data.
+        MetaTransactionDataV2 mtx;
+        // The meta-transaction signature (by `mtx.signer`).
+        LibSignature.Signature signature;
+        // The selector of the function being called.
+        bytes4 selector;
+        // The ETH balance of this contract before performing the call.
+        uint256 selfBalance;
+        // The block number at which the meta-transaction was executed.
+        uint256 executedBlockNumber;
+    }
+
+    /// @dev Arguments for a `TransformERC20.transformERC20()` call.
+    struct ExternalTransformERC20Args {
+        IERC20Token inputToken;
+        IERC20Token outputToken;
+        uint256 inputTokenAmount;
+        uint256 minOutputTokenAmount;
+        ITransformERC20Feature.Transformation[] transformations;
+    }
+
+    /// @dev Name of this feature.
+    string public constant override FEATURE_NAME = "MetaTransactionsV2";
+    /// @dev Version of this feature.
+    uint256 public immutable override FEATURE_VERSION = _encodeVersion(1, 2, 1);
+    /// @dev EIP712 typehash of the `MetaTransactionData` struct.
+    bytes32 public immutable MTX_EIP712_TYPEHASH =
+        keccak256(
+            "MetaTransactionDataV2("
+            "address signer,"
+            "address sender,"
+            "uint256 expirationTimeSeconds,"
+            "uint256 salt,"
+            "bytes callData,"
+            "address feeToken,"
+            "uint256 feeAmount,"

```

Comment:
`feeAmount` is not a member of this struct

Reply:
This issue should be fixed in the latest version.